qrcode/compat/png.py

```diff
@@ -1,7 +1,7 @@
+import contextlib
+
 # Try to import png library.
 PngWriter = None
 
-try:
-    from png import Writer as PngWriter  # type: ignore  # noqa: F401
-except ImportError:  # pragma: no cover
-    pass
+with contextlib.suppress(ImportError):
```

Comment:
`contextlib.suppress` is much slower at runtime. It should be used sparingly.

I personally never enable that `SIM` rule.